src/aspire/utils/blk_diag_matrix.py

```diff
@@ -860,7 +861,7 @@ def filter_to_fb_mat(h_fun, fbasis):
         rmat = 2*k_vals.reshape(n_k, 1)*fbasis.r0[0:k_max, ell].T
         fb_vals = np.zeros_like(rmat)
         ind_radial = np.sum(fbasis.k_max[0:ell])
-        fb_vals[:, 0:k_max] = fbasis._precomp['radial'][:,
+        fb_vals[:, 0:k_max] = radial[:,
```

Comment:
Can we keep this on one line? If not, can we break it in a more natural spot?

Reply:
I tried to limit the line to below 76 characters without using `\` for continuing. Changed to one line in new commit.

Reply:
Yes but you can break before the `*` by enclosing in parentheses instead of breaking in the middle of the indices.

Reply:
If breaking at  `*` after radial array,  Pycharm will automatically add `\` to make the next line as continuing one without ambiguity.  Anyway, only use one line in the new version. We might use some tool to limit all code to the style of 76 characters if we really need it in future.